nucliadb_utils/nucliadb_utils/settings.py

```diff
@@ -218,4 +218,4 @@ class NATSConsumerSettings(BaseSettings):
 
 
 def is_onprem_nucliadb() -> bool:
-    return nuclia_settings.nuclia_service_account is not None
```

Comment:
careful with this change. I know using nuclia_service_account is not explicit, but...

Is onprem set to false by default? If so, Is it set to true on all forms of onprem deployments?
Otherwise -- if it's True by default -- is it set to false via values on all hosted deployments?